website/pages/docs/nia/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+layout: docs
+page_title: Network Infrastructure Automation <sup>Tech Preview</sup>
+sidebar_title: Network Infrastructure Automation <sup>Tech Preview</sup>
+description: >-
+  Network Infrastructure Automation (NIA) relies on a declarative, workflow and service driven network automation architecture. Consul-Terraform-Sync is a daemon based architecture that utilizes Terraform as the underlying automation tool and leverages the Terraform provider ecosystem to drive relevant change to the network infrastructure. This usage of the Terraform provider in conjunction with Consul allows for Consul to act as the core data source to inform updates to the state of the infrastructure.
```

Comment:
Consider expanding the first couple of sentences to bridge the gap between the concept and the tool. Maybe something like
"Services registered in Consul can also be used as the source of truth to feed updates to network infrastructure like load balancers and firewalls.  Network Infrastructure Automation (NIA) relies on a declarative, workflow and service driven network automation architecture. Consul-Terraform-Sync is a new daemon based tool that utlitizes Terraform as.."

Reply:
I updated this but different from the suggestion. It's now - "Network Infrastructure Automation (NIA) is the concept of dynamically updating infrastructure devices triggered by service changes. Consul-Terraform-Sync is a tool that performs NIA and utilizes Consul as a data source that contains networking information about services and monitors those services. Terraform is used as the underlying automation tool and leverages the Terraform provider ecosystem to drive relevant changes to the network infrastructure."